src/arvfeatures.h.in

```diff
@@ -47,4 +47,12 @@
 
 #define ARAVIS_HAS_PACKET_SOCKET @ARAVIS_HAS_PACKET_SOCKET@
 
+
+#define ARAVIS_HAS_FAST_HEARTBEAT @ARAVIS_HAS_FAST_HEARTBEAT@
```

Comment:
This code change is not related to the pull request

Reply:
Sorry - I have committed to the same branch changes related to two different pull requests